boards/posix/nrf_bsim/Kconfig.defconfig

```diff
@@ -81,6 +81,26 @@ choice LOG_MODE
 	default LOG_MODE_IMMEDIATE
 endchoice
 
+# Since flash & ram is "free", we can also include the function names
+# in all logging levels.
+config LOG_FUNC_NAME_PREFIX_ERR
```

Comment:
I'm ok with this changes, but others who use this target quite a bit may want to chip in.
@Emil-Gydesen-Bose @kruithofa @cvinayak @alxelax @PavelVPV

Reply:
I don't think it does. it's only enabled for LOG_DBG like the rest of zephyr.
This also adds the thread name which is super useful when you have multiple contexts running the same fns